### ISSUE TITLE
Uncaught TypeError: validateEventDispatches is not a function

### ISSUE THREAD

**Issue by johntdyer** [lloiser/go-debug#29]
[Enter steps to reproduce below:]

1. Open up Atom Beta on OSX ( 1.8.0-beta1 )
2. watch app keep prompting about type error

**Atom Version**: 1.8.0-beta1
**System**: Mac OS X 10.11.4
**Thrown From**: [go-debug](https://github.com/lloiser/go-debug) package, v0.1.12

### Stack Trace

Uncaught TypeError: validateEventDispatches is not a function

```
At /Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/EventPluginUtils.js:99

TypeError: validateEventDispatches is not a function
    at Object.executeDispatchesInOrder (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/EventPluginUtils.js:99:5)
    at executeDispatchesAndRelease (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/EventPluginHub.js:42:22)
    at executeDispatchesAndReleaseTopLevel (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/EventPluginHub.js:53:10)
    at Array.forEach (native)
    at forEachAccumulated (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/forEachAccumulated.js:24:9)
    at Object.EventPluginHub.processEventQueue (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/EventPluginHub.js:215:7)
    at runEventQueueInBatch (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventEmitterMixin.js:18:18)
    at Object.ReactEventEmitterMixin.handleTopLevel [as _handleTopLevel] (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventEmitterMixin.js:29:5)
    at handleTopLevelImpl (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:73:24)
    at ReactDefaultBatchingStrategyTransaction.Mixin.perform (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/Transaction.js:136:20)
    at Object.ReactDefaultBatchingStrategy.batchedUpdates (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactDefaultBatchingStrategy.js:63:19)
    at Object.batchedUpdates (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactUpdates.js:97:20)
    at ReactEventListener.dispatchEvent (/Users/jdyer/.atom/packages/go-debug/node_modules/react-for-atom/node_modules/react/lib/ReactEventListener.js:150:20)
```

### Commands

```
     -2:22.1.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -1:55.1.0 core:close (div.panels-item)
  2x -1:46.6.0 find-and-replace:select-next (atom-text-editor.editor.is-focused)
     -1:45 editor:duplicate-lines (atom-text-editor.editor.is-focused)
  2x -1:44 editor:delete-to-end-of-subword (atom-text-editor.editor.is-focused)
     -1:39.4.0 editor:duplicate-lines (atom-text-editor.editor.is-focused)
     -1:16.9.0 settings-view:open (atom-text-editor.editor.is-focused)
     -1:11.2.0 core:select-all (atom-text-editor.editor.mini.is-focused)
     -0:47.8.0 go-debug:stop (atom-text-editor.editor.is-focused)
     -0:04.8.0 go-debug:toggle-panel (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
atom-prism, v0.1.2
autocomplete-go, v1.1.0
builder-go, v1.0.0
dockerletion, v0.2.0
environment, v1.2.0
go-config, v1.1.4
go-debug, v0.1.12
go-get, v1.0.3
go-plus, v4.1.0
gofmt, v1.1.6
gometalinter-linter, v1.1.0
gorename, v1.0.2
language-docker, v1.1.6
linter, v1.11.4
linter-docker, v0.1.2
linter-golinter, v1.2.0
navigator-godef, v1.0.3
pretty-json, v1.2.0
tester-go, v1.0.4

# Dev
No dev packages
```

**Comment by lloiser** [comment]
seems to be the same problem as in https://github.com/facebook/nuclide/issues/463 ...
unfortunately no solution yet ...

**Comment by ssorallen** [comment]
[react-for-atom v15.0.2-1](https://github.com/jgebhardt/react-for-atom/tree/v15.0.2-1) fixed https://github.com/facebook/nuclide/issues/463. Try updating this package to v15.0.2-1 and see if this problem persists.

**Comment by lloiser** [comment]
seems to be fixed with the new react version! (a4e227bc692114afec8a1881f99535f3c541e94e)
